commands/bootstrap/resources.go

```diff
@@ -86,7 +86,6 @@ func extractResources(resourcesZipFilePath, resourcesDir string) error {
 			if err != nil {
 				return err
 			}
-			defer dest.Close()
```

Comment:
Why did you removed the close? if io.Copy errors `dest`will not be closed...

Reply:
I removed the defer from the loop. The close is below

Reply:
But yes, it's a bad idea

Reply:
I missed the loop when reviewing...
If there is a defer in a loop that sounds like it should be done into a function (even an anonymous one)